<!DOCTYPE html>
<html><head><title>_clientDiagnostics</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">1 write to _clientDiagnostics</div><div class="rA">Azure.ResourceManager.Network (1)</div><div class="rG" id="Azure.ResourceManager.Network"><div class="rF"><div class="rN">Generated\PrivateEndpointsRestOperations.cs (1)</div>
<a href="../Generated/PrivateEndpointsRestOperations.cs.html#42"><b>42</b><i>_clientDiagnostics</i> = clientDiagnostics;</a>
</div>
</div>
<div class="rH">14 references to _clientDiagnostics</div><div class="rA">Azure.ResourceManager.Network (14)</div><div class="rG" id="Azure.ResourceManager.Network"><div class="rF"><div class="rN">Generated\PrivateEndpointsRestOperations.cs (14)</div>
<a href="../Generated/PrivateEndpointsRestOperations.cs.html#90"><b>90</b>throw await <i>_clientDiagnostics</i>.CreateRequestFailedExceptionAsync(message.Response).ConfigureAwait(false);</a>
<a href="../Generated/PrivateEndpointsRestOperations.cs.html#119"><b>119</b>throw <i>_clientDiagnostics</i>.CreateRequestFailedException(message.Response);</a>
<a href="../Generated/PrivateEndpointsRestOperations.cs.html#175"><b>175</b>throw await <i>_clientDiagnostics</i>.CreateRequestFailedExceptionAsync(message.Response).ConfigureAwait(false);</a>
<a href="../Generated/PrivateEndpointsRestOperations.cs.html#208"><b>208</b>throw <i>_clientDiagnostics</i>.CreateRequestFailedException(message.Response);</a>
<a href="../Generated/PrivateEndpointsRestOperations.cs.html#264"><b>264</b>throw await <i>_clientDiagnostics</i>.CreateRequestFailedExceptionAsync(message.Response).ConfigureAwait(false);</a>
<a href="../Generated/PrivateEndpointsRestOperations.cs.html#297"><b>297</b>throw <i>_clientDiagnostics</i>.CreateRequestFailedException(message.Response);</a>
<a href="../Generated/PrivateEndpointsRestOperations.cs.html#342"><b>342</b>throw await <i>_clientDiagnostics</i>.CreateRequestFailedExceptionAsync(message.Response).ConfigureAwait(false);</a>
<a href="../Generated/PrivateEndpointsRestOperations.cs.html#369"><b>369</b>throw <i>_clientDiagnostics</i>.CreateRequestFailedException(message.Response);</a>
<a href="../Generated/PrivateEndpointsRestOperations.cs.html#405"><b>405</b>throw await <i>_clientDiagnostics</i>.CreateRequestFailedExceptionAsync(message.Response).ConfigureAwait(false);</a>
<a href="../Generated/PrivateEndpointsRestOperations.cs.html#425"><b>425</b>throw <i>_clientDiagnostics</i>.CreateRequestFailedException(message.Response);</a>
<a href="../Generated/PrivateEndpointsRestOperations.cs.html#470"><b>470</b>throw await <i>_clientDiagnostics</i>.CreateRequestFailedExceptionAsync(message.Response).ConfigureAwait(false);</a>
<a href="../Generated/PrivateEndpointsRestOperations.cs.html#502"><b>502</b>throw <i>_clientDiagnostics</i>.CreateRequestFailedException(message.Response);</a>
<a href="../Generated/PrivateEndpointsRestOperations.cs.html#542"><b>542</b>throw await <i>_clientDiagnostics</i>.CreateRequestFailedExceptionAsync(message.Response).ConfigureAwait(false);</a>
<a href="../Generated/PrivateEndpointsRestOperations.cs.html#569"><b>569</b>throw <i>_clientDiagnostics</i>.CreateRequestFailedException(message.Response);</a>
</div>
</div>
</body></html>